browser/ui/webui/brave_new_tab_ui.cc

```diff
@@ -41,6 +44,17 @@ class NewTabDOMHandler : public content::WebUIMessageHandler {
   DISALLOW_COPY_AND_ASSIGN(NewTabDOMHandler);
 };
 
+bool IsQwantUsedInQwantRegion(Profile* profile) {
+  bool region_for_qwant =
+      TemplateURLPrepopulateData::GetPrepopulatedDefaultSearch(
+          profile->GetPrefs())->prepopulate_id ==
+          TemplateURLPrepopulateData::PREPOPULATED_ENGINE_ID_QWANT;
+  bool qwant_used =
+      profile->GetPrefs()->GetInteger(kAlternativeSearchEngineProviderInTor) ==
+      TemplateURLPrepopulateData::PREPOPULATED_ENGINE_ID_QWANT;
```

Comment:
`qwant_used` only returns true for Tor tabs with this logic; this should return true also for incognito windows